module/observations/helper.js

```diff
@@ -1048,4 +1052,179 @@ module.exports = class ObservationsHelper {
         });
     }
 
+
+         /**
+      * observation link.
+      * @method
+      * @name getObservationLink
+      * @param  {String} observationSolutionId observation solution external Id.
+      * @param  {String} appName name of app.
+      * @returns {getObservationLink} observation getObservationLink.
+     */
+
+    static getObservationLink(observationSolutionId, appName) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                if (!observationSolutionId || observationSolutionId == "" || !appName || appName == "") {
```

Comment:
@priyanka-TL  Do the check of query,params and body from the request inside validator file which will be present inside each module file.